### PR TITLE
general updates to 5.1.  manafication fix

### PR DESCRIPTION
**Pull request by Cephid**
basic updates for 5.1.

### CODE REVIEW

File: src/data/ACTIONS/RDM.js [hunk 1]
```diff
@@ -258,7 +258,7 @@ export default {
 		onGcd: true,
 		cooldown: 2.5,
 		castTime: 2,
-		potency: 270, //Increase Black by 9
+		potency: 350, //Increase Black by 9
```
Comment (Toastdeib) [inline src/data/ACTIONS/RDM.js]: Dunno if it matters for analysis, but for correctness - per garland, it looks like you got Fire/Stone and Aero/Thunder flipped, potency-wise (Fire/Stone should be 300, Aero/Thunder should be 350).

File: src/parser/jobs/rdm/modules/Gauge.js [hunk 2]
```diff
@@ -247,7 +247,7 @@ export default class Gauge extends Module {
 			this._blackManaLostToInvulnerable += gaugeAction.mana.black.invulnLoss
 
 			this._whiteManaLostToManafication += gaugeAction.mana.white.manaficationLoss
-			this._blackManaWastedToManafication += gaugeAction.mana.black.manaficationLoss
+			this._blackManaLostToManafication += gaugeAction.mana.black.manaficationLoss
```
Comment (Toastdeib) [inline src/parser/jobs/rdm/modules/Gauge.js]: RIP.

File: src/parser/jobs/rdm/index.js [hunk 3]
```diff
@@ -21,7 +21,7 @@ export default new Meta({
 	</>,
 	supportedPatches: {
 		from: '5.05',
-		to: '5.08',
+		to: '5.1',
```
Comment (ackwell) [inline src/parser/jobs/rdm/index.js]: are these changes gonna work on the old patch?

i know there's not much tooling for multiple patches worth of the DATA file, but it still has an effect.

Comment (Cephid) [reply src/parser/jobs/rdm/index.js]: Acceleration uptime would be incorrect for older patches.  I didn't think about the from patch list.  I'll update it to 5.1 to 5.1

File: src/data/ACTIONS/RDM.js (hunk 1 above)
Comment (Cephid) [reply src/data/ACTIONS/RDM.js]: Indeed I did, I will correct it.

**Pull request merged**